v3.0/reference/sql/functions-and-operators/expressions-pushed-down.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 下推到 TiKV 的表达式列表
+category: reference
+---
+
+# 下推到 TiKV 的表达式列表
+
```

Comment:
Add a brief intro here to this whole doc.

Reply:
Added.